cmd/template/cluster/provider/templates/aws/kubeadm_control_plane.yaml.tmpl

```diff
@@ -0,0 +1,65 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+kind: KubeadmControlPlane
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .ReleaseVersion }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Owner }}"
+    "cluster.x-k8s.io/watch-filter": "capi"
+  name: {{ .Name }}-control-plane
+  namespace: {{ .Namespace }}
+spec:
+  infrastructureTemplate:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AWSMachineTemplate
+    name: {{ .Name }}-control-plane
+  kubeadmConfigSpec:
+    clusterConfiguration:
+      apiServer:
+        extraArgs:
+          cloud-provider: aws
+      controllerManager:
+        extraArgs:
+          cloud-provider: aws
+    files:
+    - content: IyBVc2UgbW9zdCBkZWZhdWx0cyBmb3Igc3NoZCBjb25maWd1cmF0aW9uLgpTdWJzeXN0ZW0gc2Z0cCBpbnRlcm5hbC1zZnRwCkNsaWVudEFsaXZlSW50ZXJ2YWwgMTgwClVzZUROUyBubwpVc2VQQU0geWVzClByaW50TGFzdExvZyBubyAjIGhhbmRsZWQgYnkgUEFNClByaW50TW90ZCBubyAjIGhhbmRsZWQgYnkgUEFNCiMgTm9uIGRlZmF1bHRzICgjMTAwKQpDbGllbnRBbGl2ZUNvdW50TWF4IDIKUGFzc3dvcmRBdXRoZW50aWNhdGlvbiBubwpUcnVzdGVkVXNlckNBS2V5cyAvZXRjL3NzaC90cnVzdGVkLXVzZXItY2Eta2V5cy5wZW0KTWF4QXV0aFRyaWVzIDUKTG9naW5HcmFjZVRpbWUgNjAKQWxsb3dUY3BGb3J3YXJkaW5nIG5vCkFsbG93QWdlbnRGb3J3YXJkaW5nIG5vCg==
+      encoding: base64
+      owner: root
+      path: /etc/ssh/sshd_config
```

Comment:
I'm concerned about how many places things like this (which IIUC likely won't be re-defaulted) will need to be kept in sync. I could see us easily updating a value in `happa` but not here, for example. If we can't extract the templates to a single separate location (to avoid fetching them dynamically? I forget the reason), maybe this can become the single source of truth?

Reply:
this is just a temporary place to be able to work with capa,  if we solve the issue we currently have with ssh and rsa key we could remove the config  here

and once kyverno can do mutating  on secrets we could drop most of the content for bastion completely

Reply:
I think we can work a lot to move things to webhooks / central config. I would like to keep this PR about the client implementation for now :)